### PR TITLE
Emergency space suit in emergency lockers

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
in response to this pr: https://github.com/tgstation/tgstation/pull/85706
basically the emergency suit from the emergency pods.
Has a 50% chance of getting torn on getting attacked, making it useless.
Has an upgraded cell which lasts for roughly 137 seconds of continuous use
suit case sprite taken from https://github.com/NovaSector/NovaSector
## Why It's Good For The Game
Crew needs a cheap alternative to survive mass spacings, without it, survivability of crew will plummet to extremely low levels, which isn't very fun for anyone as PVP interaction is much more interesting than PVE hazards.
It's still a very weak space suit, but at least it will save you from low pressure.
## Changelog
:cl: grungussuss
balance: emergency space suits are now included in emergency oxygen closets
image: added the sprite for the emergency space suit case
/:cl:

### CODE REVIEW

**Comment by Sadboysuss** [comment]
> in response to this pr: #85706

> in response to this pr: https://github.com/tgstation/tgstation/pull/85706

Read pr body

**Comment by unit0016** [comment]
Incredibly funny we've circled back around to beestation skinsuits again; but this PR incidentally revealed to me we have THREE different; near identical; orange softsuits:

- /obj/item/clothing/suit/space/orange and /obj/item/clothing/head/helmet/space/orange; which are identical to a normal softsuit minus minus slowdown
- /obj/item/clothing/head/helmet/space/fragile and /obj/item/clothing/suit/space/fragile, which have less slowdown but a chance to tear when attacked
- and /obj/item/clothing/head/helmet/space/syndicate/orange and /obj/item/clothing/suit/space/syndicate/orange; which have the syndicate softsuit armor stats.

Shit's weird. Not your fault at all; just weird.

**Comment by Exester509** [comment]
I was gonna suggest it not having a temperature regulator so it's actually just for emergencies, but plasmamen are cold immune so that'd just make it more powergamey for the race. The suit being torn is pretty funny and effective at dissuading powergame

**Comment by Sadboysuss** [comment]
> I was gonna suggest it not having a temperature regulator so it's actually just for emergencies, but plasmamen are cold immune so that'd just make it more powergamey for the race. The suit being torn is pretty funny and effective at dissuading powergame

a much better solution would be to give it a shitty cell, but I don't think it's a problem in the first place

File: code/modules/shuttle/emergency.dm [hunk 1]
```diff
@@ -797,6 +797,27 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/item/storage/pod, 32)
...
+/obj/item/storage/box/emergency_spacesuit
+	name = "emergency space suit case"
+	desc =  "A small case containing an emergency space suit and helmet."
+	icon = 'icons/obj/storage/storage.dmi'
+	icon_state = "emergency_spacesuit_box"
+	illustration = null
+
+/obj/item/storage/box/emergency_spacesuit/Initialize(mapload)
+	. = ..()
+	w_class = WEIGHT_CLASS_BULKY
```
Comment (Watermelon914) [inline code/modules/shuttle/emergency.dm:809]: this can be set on the type itself, doesn't need to be set on init

Comment (Sadboysuss) [reply code/modules/shuttle/emergency.dm:809]: oh yeah my bad

**Comment by Higgin** [comment]
finally.

**Comment by Mothblocks** [comment]
feels weird for this to be in virtually *every* locker

File: code/modules/shuttle/emergency.dm [hunk 2]
```diff
@@ -797,6 +797,27 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/item/storage/pod, 32)
...
+	illustration = null
+
+/obj/item/storage/box/emergency_spacesuit/Initialize(mapload)
+	. = ..()
+	atom_storage.max_specific_storage = WEIGHT_CLASS_BULKY
+	atom_storage.max_slots = 2
+	atom_storage.set_holdable(list(
+		/obj/item/clothing/head/helmet/space/fragile,
+		/obj/item/clothing/suit/space/fragile,
+		))
```
Comment (Mothblocks) [inline code/modules/shuttle/emergency.dm:815]:
```suggestion
	))
```

**Comment by Sadboysuss** [comment]
> feels weird for this to be in virtually _every_ locker

can put it to chance if there is one

**Pull request closed without merging**